src/dependency-manager/graph/node/service.ts

```diff
@@ -43,4 +43,13 @@ export class ServiceNode extends DependencyNode implements ServiceNodeOptions {
   get is_local(): boolean {
     return this.local_path !== '';
   }
+
+  get architect_ref(): string {
```

Comment:
Can we abstract this logic into a util function?

Reply:
Updated by adding a function to `src/dependency-manager/utils/refs.ts`

Reply:
Updated by adding a function to `src/dependency-manager/utils/refs.ts`